.vscode/settings.json

```diff
@@ -18,5 +23,6 @@
   ],
   "files.watcherExclude": {
     "**/target": true
-  }
-}
+  },
+  "cmake.configureOnOpen": false
```

Comment:
Is this something project-specific? I don't think we are using cmake anywhere.

Reply:
VS code just keeps pestering me about setting up CMake without this. Not sure why it is so keen on this.